### PR TITLE
Fix the leftover master to main issues

### PR DESCRIPTION
**Pull request by reyang**
Follow up #1731.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -186,12 +186,11 @@ issue](https://github.com/open-telemetry/opentelemetry-specification/issues/165)
...
-[`.editorconfig`](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/.editorconfig)
-file which is supported by all the IDEs/editor mentioned above. It works with
-the IDE/editor only and does not affect the actual build of the project.
+This project includes a [`.editorconfig`](./.editorconfig) file which is
+supported by all the IDEs/editor mentioned above. It works with the IDE/editor
+only and does not affect the actual build of the project.
 
 This repository also includes [stylecop ruleset
-files](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/build).
+files](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/build).
```
Comment (cijothomas) [inline CONTRIBUTING.md:194]: relative link?

Comment (reyang) [reply CONTRIBUTING.md:194]: docfx doesn't support relative link to a folder (it only supports link to a file) - so if we change it to relative, we'll get a build error.

Comment (reyang) [reply CONTRIBUTING.md:194]: I could change this to one file (e.g. `OpenTelemetry.prod.ruleset`) if it sounds okay.

Comment (reyang) [reply CONTRIBUTING.md]: I just changed it to a text instead of link and it seems better.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**